src/css/components/_play-pause.scss

```diff
@@ -1,3 +1,6 @@
+.video-js .vjs-play-control {
+  cursor: pointer;
+}
 .video-js .vjs-play-control .vjs-icon-placeholder {
   cursor: pointer;
```

Comment:
this line is probably no longer necessary.

Reply:
Ah, you are right. Fixed now.